manifests/cf-manifest/deployments/900-cf-tests.yml

```diff
@@ -41,6 +41,8 @@ properties:
     include_operator: true
     include_services: true
     include_route_services: false
+    default_timeout: 500
+    nodes: 6
```

Comment:
Was this committed by accident? No mention in the commit message and I think it should be handled in [#113369495](https://www.pivotaltracker.com/story/show/113369495). Could you leave a comment there about whether this proved to be effective?